cts.schema.json

```
@@ -112,7 +122,19 @@
       "type": "boolean",
       "const": true,
       "description": "The flag indicating that the selector is not a valid JSONPath selector expression"
+    },
+    "result_paths": {
+      "description": "The expected normalized JSONPath query strings, one for each result value",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "results_paths": {
+      "description": "An array of possible expected result paths",
```

I think that description is a little unclear.
```suggestion
      "description": "An array of possible expected normalized JSONPath query strings of applying the selector to the document, each element of which contains all the normalized JSONPath query strings of matched values",
```